app/utils.py

```diff
@@ -63,6 +63,15 @@ def is_app_closed():
     return False
 
 
+def is_online_checkin_closed():
+    opens = getattr(settings, 'ONLINE_CHECKIN', None)
+    if opens:
+        time_left = timezone.now() - opens
```

Comment:
I don't understand this, pero hauria de estar actiu el divendres a partir de les 17:00 fins disabte si vols tmb a les 17:00. podries fer que l'hora actual estigui entre la ONLINE_CHECKIN i la ONLINE_CHECKIN + 1 dia, pls? Esque els dires volen divendres i dissabte, i crec que aixo em sembla el mes facil de fer.